test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
"most likely" is the keyword here.

That's probably not the best place to discuss about that, but I've written tons of methods where authentication was not mandatory but if the user was authenticated, the payload was different (for instance, a profile endpoint: if the request is anonymous, you get a minimal representation. If you're connected but not the user corresponding to the profile, you get a intermediate representation... and you get the full profile it it's yours)

@blowdart I agree. Even if we could argue that social providers should be configured to manual mode, you should have a way to force the automatic/active mode, just like you could with Katana 3.
https://github.com/aspnet/Security/issues/154#issuecomment-77431018